back-end/tests/test_train.py

```diff
@@ -1,4 +1,149 @@
+import os
+import time
+
+import torch
+
 from test_app import app, client
 
 
-# class TestTrain:
\ No newline at end of file
+# class TestTrain:
```

Comment:
Maybe try to remove the large chunk of commented code next time before pushing even though we planned to fix it shortly in the next PR. This is because we may get interrupted and never fix this.